### DISCUSSION TITLE
Is there a way to rate-limit just one dependency?

### DISCUSSION THREAD

**Discussion by markelliot** [renovatebot/renovate#17492] [Request Help] [answered]
We have a Bazel dependency on https://github.com/googleapis/googleapis by its commit hash. There's no release cadence for googleapis, and commits are frequent (many every day Monday-Friday). I'd love to be able to configure a rate-limiter for just this one dependency (or for any dependency that doesn't have a more typical release/versioning process). Is there a way to configure a rate-limiter or time window for just one dependency?

**Comment by rarkins** [comment, marked as answer]
You can configure `schedule` per dependency, using `packageRules`
